workflow-templates/dependabot_approve_and_label.yml

```diff
@@ -42,13 +39,14 @@ jobs:
           echo "is_interesting_package (merge) : ${{ steps.parse_dependabot_pr_automerge.outputs.is_interesting_package }}"
           echo "is_interesting_package (release) : ${{ steps.parse_dependabot_pr_autorelease.outputs.is_interesting_package }}"
           echo "semver_increment : ${{ steps.parse_dependabot_pr_automerge.outputs.semver_increment }}"
-  approve_and_label:
+
+  approve:
     runs-on: ubuntu-latest
     needs: evaluate_dependabot_pr
-    name: Approve and label
+    name: Approve
```

Comment:
A bit of a nitpick, but maybe we should add display names to the jobs to make it clearer as to what is (or isn't) happening?  So this one could be 'Approve Dependabot PR'